taxi/views.py

```diff
@@ -52,3 +93,27 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
```

Comment:
The best practice way to use form, for this class-view

Reply:
Thanks for your feedback!